synapse/cortex.py

```diff
@@ -834,6 +834,11 @@ class Cortex(s_cell.Cell):  # type: ignore
             'description': 'Logging log level to emit storm logs at.',
             'type': 'integer'
         },
+        'http:proxy': {
+            'description': 'An aiohttp-socks compatible proxy URL to use storm HTTP API.',
+            'type': 'string',
+            'hidecmdl': True,
```

Comment:
Why is this set to hidecmdl ?

Reply:
To avoid config option CLI param clutter, but i could def remove.